FAQ.md

```diff
@@ -246,6 +246,36 @@ To run and train InstructLab locally, you must meet the following requirements:
 - Approximately 60GB of free disk space is needed to run the entire process locally on Apple hardware
 - About 32 GB RAM
 
+## Glossary
+
+| Term | Explanation | Additional Reference |
+| --- | --- | --- |
+| Checkpoints | Snapshots during training. They are scored individually and the best is selected. | N/A |
+| CUDA | “Compute Unified Device Architecture” - A parallel computing platform and API for general computing on GPUs by NVIDIA. | [Ref](https://www.nvidia.com/en-us/geforce/technologies/cuda/) |
+| DeepSpeed | Deep learning optimization library for PyTorch | [Ref](https://www.deepspeed.ai/) |
+| Granite | Open source licenced LLM released by IBM | [Ref](https://huggingface.co/ibm-granite) |
```

Comment:
you can s/licenced/licensed/ here but I'd probably just remove the word in this case.  Either way.

Reply:
Good idea!

Reply:
Hmm I was always told to use "open source licensed" just as a percussion when referring to Granite